### PR TITLE
make code compatible with apispec 1.2.0

### PR DESCRIPTION
**Pull request by buxx**

### CODE REVIEW

File: tests/ext/unit/test_aiohttp.py [hunk 1]
```diff
@@ -507,6 +508,9 @@ class UserSchema(marshmallow.Schema):
         )
 
         doc = hapic.generate_doc("aiohttp", "testing")
+        # Prevent keep of OrderedDict
```
Comment (lebouquetin) [inline tests/ext/unit/test_aiohttp.py]: who is the author of the comment? what is the comment date?

File: tests/ext/unit/test_aiohttp.py [hunk 2]
```diff
@@ -573,13 +577,16 @@ class OuputStreamItemSchema(marshmallow.Schema):
...
```
Comment (lebouquetin) [inline tests/ext/unit/test_aiohttp.py]: same remark. if a comment is required, then add date+author+type of comment HACK/INFO/TODO/FIXME etc

File: tests/ext/unit/test_aiohttp.py [hunk 3]
```diff
@@ -573,13 +577,16 @@ class OuputStreamItemSchema(marshmallow.Schema):
...
+        doc = json.loads(json.dumps(doc))
+
         assert "/" in doc.get("paths")
         assert "get" in doc["paths"]["/"]
-        assert 200 in doc["paths"]["/"]["get"].get("responses", {})
+        assert '200' in doc["paths"]["/"]["get"].get("responses", {})
```
Comment (lebouquetin) [inline tests/ext/unit/test_aiohttp.py]: why is the HTTP code a string?

File: tests/func/fake_api/test_fake_api.py [hunk 4]
```diff
@@ -170,11 +172,11 @@ def test_func__test_fake_api_doc_ok__all_framework(context):
     doc = hapic.generate_doc(
         title="Fake API", description="just an example of hapic API"
     )
+    # Prevent keep of OrderedDict
```
Comment (lebouquetin) [inline tests/func/fake_api/test_fake_api.py]: same remark

File: tests/ext/unit/test_aiohttp.py (hunk 3 above)
Comment (buxx) [reply tests/ext/unit/test_aiohttp.py]: It is a apispec change (since 1.2.0)

File: tests/ext/unit/test_aiohttp.py (hunk 1 above)
Comment (buxx) [reply tests/ext/unit/test_aiohttp.py]: changed

File: tests/ext/unit/test_aiohttp.py (hunk 2 above)
Comment (buxx) [reply tests/ext/unit/test_aiohttp.py]: done

File: tests/func/fake_api/test_fake_api.py (hunk 4 above)
Comment (buxx) [reply tests/func/fake_api/test_fake_api.py]: done

**Pull request merged**